cookbooks/bcpc-hadoop/attributes/hive.rb

```diff
@@ -1,3 +1,6 @@
 default[:bach][:hive][:metastore][:port] = 9083
 default[:bach][:hive][:hiveserver2][:port] = 10000
```

Comment:
It would be good to have all the `default[:bach][:hive][:hiveserver2]` attributes together